python/ray/worker.py

```diff
@@ -623,13 +623,23 @@ def init(
 
         ray.init()
 
-    To connect to an existing Ray cluster, use this as follows (substituting
-    in the appropriate address):
+    To connect a driver to an existing Ray cluster, use this as follows
+    (substituting in the appropriate address):
 
     .. code-block:: python
 
         ray.init(address="123.45.67.89:6379")
 
+    To connect to an existing Ray cluster using Ray client, specify a protocol
```

Comment:
tbh, why do we need to know about `ray client` here? I wish there was a way we could talk about this without actually mentioning a new concept.

Reply:
True, doesn't need to be known here. Swapped out the wording to: ` To connect to an existing remote cluster, use this as follows (substituting in the appropriate address). Note the addition of "ray://" at the beginning of the address.`